docs/proxy/google_serp_proxy.md

```diff
@@ -1,47 +1,58 @@
-*   Google Shopping (`http://www.google.<country domain>/search?tbm=shop`)
+For code examples on how to connect to Google SERP proxies, see the [examples]({{@link proxy/google_serp_proxy/examples.md}}) page.
 
-**Important:** Only HTTP requests are allowed, and the Google hostname needs to start with `www.` subdomain.
+### [](#username-parameters) Username parameters
 
-**Pricing is based on the number of requests made**. Please [contact us](https://apify.com/contact) if you want to use Google SERP Proxy or if you need more information.
+The `username` field enables you to pass various [parameters]({{@link proxy/connection_settings.md#username-parameters}}), such as groups and country, for your proxy connection.
 
-## [](#username-parameters)Username parameters
+When using Google SERP proxy, the username should always be:
 
-HTTP proxy username is used to pass various parameters for the proxy connection.
+```
+groups-GOOGLE_SERP
+```
 
-In the case of Google SERP proxy, the username should always look like this
+Unlike [datacenter]({{@link proxy/datacenter_proxy.md}}) or [residential]({{@link proxy/residential_proxy.md}}) proxies, there is no [session]({{@link proxy/connection_settings.md#username-parameters}}) parameter.
 
-    groups-GOOGLE_SERP
+If you use the `country` [parameter]({{@link proxy/connection_settings.md#username-parameters}}), the Google proxy location is used if you access a website whose hostname (stripped of `www.`) starts with **google**.
 
-Unlike datacenter or residential proxies, there is no session parameter.
+## [](#country-selection) Country selection
 
-## [](#country-selection)Country selection
+You must use the correct Google domain to get results for your desired country code.
 
-A correct google domain needs to be used to get results for the desired country code.
 For example:
 
-Search results from the US: `http://www.google.com/search?q=<query>`
-
-Shopping results from Great Britain: `http://www.google.co.uk/seach?tbm=shop&q=<query>`
+* Search results from the USA: `http://www.google.com/search?q=<query>`
 
-Search results from Germany: `http://www.google.de/search?q=<query>`
+* Shopping results from Great Britain: `http://www.google.co.uk/seach?tbm=shop&q=<query>`
 
-See [full list of available domain names](https://ipfs.io/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/wiki/List_of_Google_domains.html) for specific countries. When used, always remember to prepend the domain name with `www.` prefix.
+See a [full list](https://ipfs.io/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/wiki/List_of_Google_domains.html) of available domain names for specific countries. When using them, remember to prepend the domain name with the `www.` prefix.
 
+## [](#using-with-puppeteer) Using with Puppeteer
 
+Google SERP proxy cannot be used with [Puppeteer](https://pptr.dev/), due to HTTP Strict Transport Security [headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security).
```

Comment:
Ummm, even I don't know what this means 😄 Does that mean that Chromium/Chrome would enforce a `https` connection to Google which is not supported by the proxy?

Reply:
Damn, I have no idea where I got this from 😕 Doesn't even make sense, since we have examples on how to use PuppeteerCrawler with SERP. 

I think this one's a total faux pas, please forgive 😄